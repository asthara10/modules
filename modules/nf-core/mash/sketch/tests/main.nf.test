nextflow_process {

    name "Test Process MASH_SKETCH"
    script "../main.nf"
    process "MASH_SKETCH"

    tag "modules"
    tag "modules_nfcore"
    tag "mash"
    tag "mash/sketch"

    test("test_mash_sketch") {


        when {
            process {
                """
                
                input[0] = [ [ id:'test', single_end:false], // meta map
              [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true),
                file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true)
              ]
            ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
