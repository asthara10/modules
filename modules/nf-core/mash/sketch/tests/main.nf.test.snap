{
    "versions": {
        "content": [
            [
                "versions.yml:md5,9c9d87626dd77c2dd8463d60357bde63"
            ]
        ],
        "timestamp": "2023-11-24T13:33:03.571313"
    }
}